File: CSharp/HtmlWalker/Exception/TagNotSupportedException.cs
using HtmlWalker.Core;

namespace HtmlWalker.Exception
{
    public class TagNotSupportedException : HtmlWalkerException
    {
        public TagNotSupportedException(string msg) : base(msg) { }

        public TagNotSupportedException(ITag adapter) : base(adapter.Name) { }
    }
}

File: Java/src/htmlwalker/exception/TagNotSupportedException.java
package htmlwalker.exception;

import htmlwalker.ITag;

@SuppressWarnings("serial")
public class TagNotSupportedException extends HtmlWalkerException
{
	
    public TagNotSupportedException(String msg) { super(msg); }

    public TagNotSupportedException(ITag adapter) { super(adapter.name()); }
    
}
